### DISCUSSION TITLE
Compiled Release

### DISCUSSION THREAD

**Discussion by Spectroxx** [fossmium/OneDrive-Cloud-Player#53] [Q&A] [answered]
Would it be possible to have a release that is already compiled?

**Comment by TimGels** [comment, marked as answer]
> 
> 
> Would it be possible to have a release that is already compiled?

I don't think it will be much of an issue to release the binaries. But certification of the UWP package for sideloading is another story, so it would probably be an unsigned package. We will look into it when we have some spare free time on our hands.

[Comment explaining how to use our self-signed certificate](https://github.com/TimGels/OneDrive-Cloud-Player/discussions/53#discussioncomment-600748)

**Reply by Spectroxx** [reply]
That's awesome! I look forward to it.

**Reply by JohannesKauffmann** [reply]
Instructions for installing the app using our self-signed certificate can now be found on the [wiki](https://github.com/TimGels/OneDrive-Cloud-Player/wiki/Installing-the-self-signed-version-of-OneDrive-Cloud-Player). Please let us know if there are any issues during installing.

**Reply by Spectroxx** [reply]
Thank you! Worked perfectly!
